test/libethereum/blockchain.cpp

```diff
@@ -56,18 +56,23 @@ void doBlockchainTests(json_spirit::mValue& _v, bool _fillin)
 
 		BOOST_REQUIRE(o.count("pre"));
 		ImportTest importer(o["pre"].get_obj());
-		TransientDirectory td_stateDB;
 		TransientDirectory td_stateDB_tmp;
-		State state(State::openDB(td_stateDB.path()), BaseState::Empty, biGenesisBlock.coinbaseAddress);
-		State stateTemp(State::openDB(td_stateDB_tmp.path()), BaseState::Empty, biGenesisBlock.coinbaseAddress);
-		importer.importState(o["pre"].get_obj(), state);
-		o["pre"] = fillJsonWithState(state);
-		state.commit();
+		State trueState(OverlayDB(State::openDB(td_stateDB_tmp.path())), BaseState::Empty, biGenesisBlock.coinbaseAddress);
+
+		//Imported blocks from the start
+		typedef std::vector<bytes> uncleList;
+		typedef std::pair<bytes, uncleList> blockSet;
+		std::vector<blockSet> blockRLPs;
```

Comment:
This name does not describe what it is. It is a vector of blockSets, so I would prefer blockSets (like Transaction and Transactions)